packages/@sanity/portable-text-editor/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
tslint is deprecated - can we use eslint instead?

Reply:
I guess we already do? Removing it.